pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "scenic"
-version = "3.0.0b1"
+version = "3.0"
```

Comment:
This should still be `3.0.0b1` (`3.0` isn't a valid semver version anyway -- apparently the patch version number is mandatory).

Reply:
Weird, I don't know why it's showing this since as far as I can tell from the git blame it went from 2.x -> 3.0 -> 3.0.0b1. Either way, it's 3.0.0b1 now.